src/main/java/rs/teslaris/core/model/commontypes/GeoLocation.java

```diff
@@ -22,4 +23,22 @@ public class GeoLocation {
 
     @Column(name = "precision_in_meters", nullable = false)
     private int precisionInMeters;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        GeoLocation that = (GeoLocation) o;
+        return precisionInMeters == that.precisionInMeters && longitude.equals(that.longitude) &&
```

Comment:
precesionInMeters is zoom level, right? Not sure whether it should be included in checking whether two geolocations are the same. I think it depends what is the scenario. We can keep as it is at the moment, but might be changed in the future

Reply:
The idea is pulled from another project where I had a feature to submit location via the phone GPS since the default javascript method navigator.geolocation.getCurrentPosition returns accuracy, I added this into the entity as metadata just in case.

In this project, it is probably irrelevant info in GeoLocation entity.